uniqueids/models.py

```diff
@@ -0,0 +1,76 @@
+import random
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class Record(models.Model):
+    """ The source from which a registation originates.
+        The User foreignkey is used to identify the source based on the
+        user's api token.
+        write_to is the field we should write back to on the identity details
+    """
+    id = models.BigIntegerField(primary_key=True)
+    identity = models.UUIDField()
+    write_to = models.CharField(max_length=36, null=False, blank=False)
```

Comment:
`max_length=36` seems a bit like it was copied from UUID length, but it should suffice.